packages/js/components/src/table/placeholder.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/element';
+import { range } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Table from './table';
+import { QueryProps, TableHeader } from './types';
+
+type TablePlaceholderProps = {
+	// An object of the query parameters passed to the page
+	query?: QueryProps;
+	// A label for the content in this table.
+	caption: string;
+	// An integer with the number of rows to display.
+	numberOfRows?: number;
+
+	rowHeader?: number | false;
+
```

Comment:
Nitpick: it would be great to remove the extra line here.

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/36370/commits/1763376e3dbea0bd80d8b5e78f01418ec96b24d2